myapp/app/controllers/users_controller.rb

```diff
@@ -23,7 +24,9 @@ def create
 
     respond_to do |format|
       if @user.save
-        format.html { redirect_to user_url(@user), notice: 'User was successfully created.' }
+        format.html do
+          redirect_to user_url(@user), flash: { success: I18n.t('messages.create', model_name: I18n.t('activerecord.models.user')) }
```

Comment:
[FYI]
モデル名に関してはこんな感じでも出せますー！

```suggestion
          redirect_to user_url(@user), flash: { success: I18n.t('messages.create', model_name: @user.model_name.human)) }
```

Reply:
ご提案ありがとうございます。
いただいた内容で修正しました。
[b32c19c](https://github.com/Fablic/training/pull/1535/commits/b32c19c0105cdb2ca55e428fea26705b79ad9d40)